includes/class-newspack-ads-model.php

```diff
@@ -508,5 +512,41 @@ function( $carry, $item ) {
 			[ 0, 0 ]
 		);
 	}
+
+	/**
+	 * Get ad targeting params for the current post or archive.
+	 *
+	 * @param array $ad_unit Ad unit to get targeting for.
+	 * @return array Associative array of targeting keyvals.
+	 */
+	public static function get_ad_targeting( $ad_unit ) {
+		$targeting = [];
+
+		if ( is_singular() ) {
+			// Add the post slug to targeting.
+			$slug = get_post_field( 'post_name' );
+			if ( $slug ) {
+				$targeting['slug'] = sanitize_text_field( $slug );
+			}
+
+			// Add the category slugs to targeting.
```

Comment:
I tried this out on a post with the categories "Arts" and "Events & Concerts" - the result only includes the word "events":

```
slug=no-category&category=arts,events&ID=8076
```

Reply:
What's the category slug of your "Events & Concerts" category? If you simply renamed the Newspack default category "Events" to "Events & Concerts", it would keep the original slug of `events`, and this would therefore be the correct behavior.

Reply:
You're right - that was my slug. I created a new one with an ampersand and multiple words and the slug was assigned correctly - nothing to see here.